templates/master.erb

```diff
@@ -1,29 +1,13 @@
-# ZooKeeper URL (used for leader election amongst masters)
-ZK="<%= @zookeeper %>"
+MESOS_work_dir="<%= @work_dir %>"
+MESOS_zk="<%= @zookeeper %>"
+MESOS_quorum=<%= @quorum %>
+MESOS_ip="<%= @listen_address %>"
+MESOS_port=<%= @master_port %>
+MESOS_cluster="<%= @cluster %>"
+MESOS_log_dir="<%= @log_dir %>"
+MESOS_logbufsecs=1
+MESOS_logging_level="INFO"
 
-# Human readable name for the cluster, displayed in the webui
-CLUSTER="<%= @cluster %>"
-
-# IP address to listen on (in case that you start master
-# on this node, it will bind to this address)
-<% if @listen_address -%>
-IP="<%= @listen_address %>"
```

Comment:
In case that `@listen_address` is `undefined` we leave up to Mesos to which address it will bind. In most cases it should be same as enforcing `0.0.0.0`, though it might break some installations...